DOCS: PLGLTS-127 (#1038)
MultiSafepay/docs: content/integrations/hosted/lightspeed_app/manual.md

If the Javascript files are not loaded, all payment methods will continue to work. If you want to use our payment icons, you can find them on our [Github page](https://github.com/MultiSafepay/MultiSafepay-icons)

> To use the app a API key is required. To obtain this key you need a MultiSafepay account. You can create a test account on https://testmerchant.multisafepay.com/signup

Reviewer: ```suggestion
To use the app, an API key is required. To obtain this key you need a MultiSafepay account. You can create a test account on https://testmerchant.multisafepay.com/signup
```

You have installed and configured the plugin successfully. If you have any questions regarding the plugin, feel free to contact our Integration team at <integration@multisafepay.com>

> After installing the MultiSafepay Payments application, you can access the settings page. You will be redirected to this page automatically after the installation or you can go there by clicking on the _Go to App_ button in your Lightspeed backend when you go to your apps and select the MultiSafepay Payments app.

Reviewer: ```suggestion
After installing the MultiSafepay Payments application, you can access the settings page. You will be redirected to this page automatically after the installation. Alternatively, navigate to your apps, select the MultiSafepay Payments app and click on the _Go to App_ button.
```
